src/server/dflycmd.cc

```diff
@@ -455,7 +455,11 @@ void DflyCmd::FullSyncFb(FlowInfo* flow, Context* cntx) {
 
   if (saver->Mode() == SaveMode::SUMMARY) {
     auto scripts = sf_->script_mgr()->GetLuaScripts();
-    ec = saver->SaveHeader(scripts);
+    StringVec script_bodies;
+    for (const auto& script : scripts) {
+      script_bodies.push_back(script.second);
+    }
+    ec = saver->SaveHeader(script_bodies);
```

Comment:
You get the scripts by value here and proceed to build a new container by copying the scripts itself

Reply:
fixed